modules/web/routing/logger.go

```diff
@@ -104,7 +104,8 @@ func logPrinter(logger log.Logger) func(trigger Event, record *requestRecord) {
 		}
 		logf := logInfo
 		// lower the log level for some specific requests, in most cases these logs are not useful
-		if strings.HasPrefix(req.RequestURI, "/assets/") /* static assets */ ||
+		if 0 < status && status < 400 &&
```

Comment:
```suggestion
		if status > 0 && status < 400 &&
```
reads more intuitively to me